java/arcgis-tiledlayer-url/README.md

```diff
@@ -1,30 +1,39 @@
-# ArcGIS Tiled Layer URL
-Load an ArcGIS Tiled Layer from a URL.
+# ArcGIS tiled layer URL
 
-![Tiled Layer from URL App](arcgis-tiledlayer-url.png)
+Load an ArcGIS tiled layer from a URL.
 
-## How to use the sample
-Simply run the app.
+![Image of ArcGIS tiled layer URL](arcgis-tiledlayer-url.png)
+
+## Use case
+
+An `ArcGISTiledLayer` consumes raster tiles provided by an ArcGIS service or a tile package (.tpk & .tpkx). Dividing a raster into tiles provides the user with relevant tiles and level of detail while panning and zooming, allowing for rapid map visualization and navigation. For example, say you wanted an application with a map containing all museums in your city so that you could share this with tourists. Since only the museums would need to support being identifiable or queryable within the app, all other contextual information required to orient and direct the tourists could be sourced from a single ArcGIS tiled layer, leading to faster performance and minimizing the time to load the map.
```

Comment:
```suggestion
A tiled layer consumes raster tiles provided by an ArcGIS service or a tile package (.tpk & .tpkx). Dividing a raster into tiles provides the user with relevant tiles and level of detail while panning and zooming, allowing for rapid map visualization and navigation. For example, say you wanted an application with a map containing all museums in your city so that you could share this with tourists. Since only the museums would need to support being identifiable or queryable within the app, all other contextual information required to orient and direct the tourists could be sourced from a single ArcGIS tiled layer, leading to faster performance and minimizing the time to load the map.
```